### ISSUE TITLE
Validated RSS URL does not show any articles. 

### ISSUE THREAD

**Issue by neilt** [ViennaRSS/vienna-rss#1357]
The following feed does not work in Vienna, but does show as a valid feed with the validator.  The information shown from the various sources below does not seem consistent.

The feed: `https://www.newsmax.com/rss/Newsfront/16/`

The validation from: `https://validator.w3.org/feed/check.cgi?url=https://www.newsmax.com/rss/Newsfront/16`

```
This feed is valid, but interoperability with the widest range of feed readers could be improved by implementing the following recommendations.
line 2, column 737: item should contain a guid element (20 occurrences) [help]
... 01A" length="46282" type="image/jpeg" /></item><item><title><![CDATA[Bor ...
                                             ^
line 2, column 12612: title should not contain HTML: &#37; [help]
... Homebuyers Fall 3&#37; to 6-Month Low]]></title><pubDate>Wed, 11 Nov 202 ...
                                             ^
line 2, column 13077: Missing atom:link with rel="self" [help]
... gth="180607" type="image/jpeg" /></item></channel></rss>
                                             ^
```

Vienna reports in the Activity view:
```
Connecting to https://www.newsmax.com/rss/Newsfront/16
HTTP code 500 reported from server
internal server error
```

Vienna report in the console:

```
default	12:52:40.179327-0600	Vienna	nw_socket_handle_socket_event [C124.1:3] Socket received READ_CLOSE event
default	12:52:40.179368-0600	Vienna	Connection 124: final read 0:0, complete[Y], final[N]
default	12:52:40.179402-0600	Vienna	Connection 124: cleaning up
default	12:52:40.179436-0600	Vienna	[C124 D8AB69A9-722D-48F7-8E5C-DEAA5DADD94A www.newsmax.com:443 tcp, url hash: 2bd21f85, tls] cancel
default	12:52:40.179491-0600	Vienna	[C124 D8AB69A9-722D-48F7-8E5C-DEAA5DADD94A www.newsmax.com:443 tcp, url hash: 2bd21f85, tls] cancelled
	[C124.1 EF11A60C-8545-4FC7-9B7E-E235D87B1D62 172.16.1.10:58097<->23.198.6.35:443]
	Connected Path: satisfied (Path is satisfied), viable, interface: en0, ipv4, ipv6, dns
	Duration: 20.115s, DNS @0.000s took 0.008s, TCP @0.008s took 0.018s, TLS 1.2 took 0.048s
	bytes in/out: 4359/1066, packets in/out: 8/7, rtt: 0.017s, retransmitted bytes: 0, out-of-order bytes: 0
default	12:52:40.179655-0600	Vienna	nw_flow_disconnected [C124.1 23.198.6.35:443 cancelled socket-flow ((null))] Output protocol disconnected
default	12:52:40.179826-0600	Vienna	nw_connection_report_state_with_handler_on_nw_queue [C124] reporting state cancelled
default	12:52:40.179899-0600	Vienna	Connection 124: done
```

**Comment by neilt** [comment]
BTW, thinking it might be a server issue I did multiple tests in the validator and it never failed and also multiple tests with Vienna and it never succeeded. All during the same time frame.

**Comment by Eitot** [comment]
It appears to be a problem with the user agent that Vienna reports to the server. The server responds with that error message.

**Comment by barijaona** [comment]
I would not call this a Vienna bug…
Should we work around this or let stupid things decided for whatever reasons remain ?

**Comment by neilt** [comment]
My request would be to make something like the following work.
```
defaults write uk.co.opencommunity.vienna2 UserAgentName Bern
```
There does not need to be any UI because this is a fairly arcane need. Even Safari allows changing the user agent name.

**Comment by barijaona** [comment]
Workaround available in version 3.6.1 and later (more details available in pull request #1373)
